helm/azure-operator-resource-chart/templates/cluster.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: "provider.giantswarm.io/v1alpha1"
+kind: AzureConfig
+metadata:
+  name: "{{ .Values.clusterName }}"
+spec:
+  azure:
+    dnsZones:
+      api:
+        name: "{{ .Values.commonDomain }}"
+        resourceGroup: "{{ .Values.commonDomainResourceGroup }}"
+      etcd:
+        name: "{{ .Values.commonDomain }}"
+        resourceGroup: "{{ .Values.commonDomainResourceGroup }}"
+      ingress:
+        name: "{{ .Values.commonDomain }}"
+        resourceGroup: "{{ .Values.commonDomainResourceGroup }}"
+    location: "{{ .Values.azure.location }}"
+    masters:
+    - vmSize: "{{ .Values.azure.vmSizeMaster }}"
+    workers:
+    - vmSize: "{{ .Values.azure.vmSizeWorker }}"
+    - vmSize: "{{ .Values.azure.vmSizeWorker }}"
+    virtualNetwork:
+      calicoSubnetCIDR: "{{ .Values.azure.calicoSubnetCIDR }}"
+      cidr: "{{ .Values.azure.cidr }}"
+      masterSubnetCIDR: "{{ .Values.azure.masterSubnetCIDR }}"
+      workerSubnetCIDR: "{{ .Values.azure.workerSubnetCIDR }}"
+  cluster:
+    calico:
+      cidr: 16
+      domain: "calico.{{ .Values.clusterName }}.{{ .Values.commonDomain }}"
+      mtu: 1500
+    customer:
+      id: "example-customer"
+    docker:
+      daemon:
+        cidr: "172.17.0.1/16"
+        extraArgs: "--log-opt max-size=25m --log-opt max-file=2 --log-opt labels=io.kubernetes.container.hash,io.kubernetes.container.name,io.kubernetes.pod.name,io.kubernetes.pod.namespace,io.kubernetes.pod.uid"
+    etcd:
+      domain: "etcd.{{ .Values.clusterName }}.{{ .Values.commonDomain }}"
+      prefix: "giantswarm.io"
+      port: 2379
+    id: "{{ .Values.clusterName }}"
+    kubernetes:
+      api:
+        altNames: "kubernetes,kubernetes.default,kubernetes.default.svc,kubernetes.default.svc.cluster.local"
+        clusterIPRange: "172.31.0.0/16"
+        domain: "api.{{ .Values.clusterName }}.{{ .Values.commonDomain }}"
+        insecurePort: 8080
+        ip: "172.31.0.1"
+        securePort: 443
+      cloudProvider: "azure"
+      dns:
+        ip: "172.31.0.10"
+      domain: "cluster.local"
+      hyperkube:
+        docker:
+          image: "quay.io/giantswarm/hyperkube:v1.8.4_coreos.0"
```

Comment:
This docker image isn't used anymore. This is now set in k8scc so we can have the k8s version in the version bundles. Can we remove it?